History/test/modal.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @fileOverview tests.js Gives testing code for the new page.
+ * This spec is managed by the Jasmine testing library.
+ *
+ * More information on testing:
+ * https://github.com/privly/privly-organization/wiki/Testing
+ *
+ **/
+
+describe ("History Application Modal Box Suite", function() {
+
+ it("does not result in an error", function() {
+
+   var $reference = jQuery('<div/>', {
+       'data-canonical-href': 'https://priv.ly/posts/2?privlyApp=PlainPost#privlyInject1',
+       'class': 'btn btn-default preview_link'
+   });
+   var iframe = jQuery('<div/>', {
+     'class': 'privly_iframe'
+   });
+   document.body.appendChild(iframe[0]);
+   historyModal.eventHandlers.iframeReturn($reference);
+
+   historyModal.eventHandlers.prevPreview();
+   historyModal.eventHandlers.nextPreview();
+   historyModal.eventHandlers.destroyLink();
+
+   privlyNetworkService.sameOriginDeleteRequest = function(){};
```

Comment:
Do you want me to comment it for you? Or would you want to do this after merging/by opening a PR?